### PR TITLE
Postgres failover slots

### PR DESCRIPTION
**Pull request by gunnarmorling**
Requires PR https://github.com/debezium/debezium/pull/6017

### CODE REVIEW

File: postgres-failover-slots/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
+services:
+  pgbouncer:
+    image: edoburu/pgbouncer:latest
+    environment:
+      - DB_HOST=postgres_replica
```
Comment (mfvitale) [inline postgres-failover-slots/docker-compose.yaml:5]: Should't this point to the `postgres_primary` at the first start?

Comment (gunnarmorling) [reply postgres-failover-slots/docker-compose.yaml:5]: Indeed it should. I had changed it while running through the steps and then committed it. Good spot!

File: postgres-failover-slots/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+     kcat -b kafka:9092 -C -o beginning -q \
+     -t dbserver1.inventory.customers | jq .payload
+
+# Modify a record in the database (current primary):
+docker run --tty --rm -i \
+     --network failover-network \
+    quay.io/debezium/tooling:1.2 \
+    bash -c 'pgcli --prompt "\u@primary:\d> " postgresql://user:top-secret@postgres_primary:5432/inventorydb'
+
+# update inventory.customers set first_name = 'Sarah' where id = 1001;
```
Comment (mfvitale) [inline postgres-failover-slots/README.md:54]: Is the comment intentional?

Comment (gunnarmorling) [reply postgres-failover-slots/README.md:54]: Yepp, it's an example of a statement to execute.

**Comment by gunnarmorling** [comment]
Thanks! Reworked one more time to assign a unique option position. All tests are passing now and this is ready to merge.

**Comment by jpechane** [comment]
@gunnarmorling Applied, thanks!

**Comment by gunnarmorling** [comment]
Woot, thanks! Is there an ETA for the release?

**Comment by jpechane** [comment]
@gunnarmorling Next week (the last before Christmas).

**Comment by gunnarmorling** [comment]
Excellent, thanks!

**Pull request merged**